common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -174,14 +174,14 @@ def username_dropdown_link_text(self):
         """
         return self.q(css='.dropdown-menu li a').text
 
-    def click_account_settings_link(self):
-        """
-        Click on `Account Settings` link.
-        """
-        self.q(css='.dropdown-menu li a').first.click()
+        def click_my_profile_link(self):
```

Comment:
@frrrances The problem is that this code is indented incorrectly. Python is strict about its whitespace, so having this at the wrong level means that it isn't declared as a function on the class. That's why the test fails, and also why the quality failed. I'm testing this locally and I'll push it up once I'm sure this is the only issue.